examples/ktor/README.md

```diff
@@ -0,0 +1,32 @@
+# Containerize a [Ktor] application with Jib
+
+This is an example of how to easily build a Docker image for a [Ktor] application with Jib.
+
+```shell
+./gradlew jibDockerBuild
+
+docker run -d --rm -p 8080:8080 ktor-jib-example
```

Comment:
Since the version is automatically set to `1` - I believe this should be `ktor-jib-example:1`

Reply:
> It seems to also tag `latest`.

I don't think this is true here. In the other Dropwizard PR, it had explicit configuration to have the additional `latest` tag, but here, I think it is not creating `latest`.

Reply:
+1 to @chanseokoh comment

For group, `example` would be fine. Only an example :)